src/components/FeatureSetsPanel/FeatureSetsPanelDataSource/FeatureSetsPanelDataSourceView.js

```diff
@@ -43,20 +51,32 @@ const FeatureSetsPanelDataSourceView = ({
             options={kindOptions}
             selectedId={data.kind}
           />
-          <Input
-            className="data-source__inputs-item"
-            floatingLabel
+          <Combobox
+            comboboxClassName="url"
+            hideSearchInput={!urlProjectItemTypeEntered}
+            inputDefaultValue={
+              data.url.pathType === MLRUN_STORAGE_INPUT_PATH_SCHEME
+                ? data.url.projectItemType
+                : ''
+            }
+            inputOnChange={path => {
+              handleUrlPathChange(path)
+            }}
+            inputPlaceholder={data.url.placeholder}
             invalid={!isUrlValid}
-            invalidText="URL is invalid"
-            label="URL"
+            invalidText={`Field must be in "${
+              pathPlaceholders[data.url.pathType]
+            }" format`}
+            matches={comboboxMatches}
+            maxSuggestedMatches={3}
             onBlur={handleUrlOnBlur}
-            onChange={handleUrlOnChange}
             required
-            requiredText="URL is required"
-            tip="For Parquet files the path could be either a file or a folder. For CSV it must be a file."
-            type="text"
-            value={data.url}
-            wrapperClassName="url"
+            requiredText="This field is required"
+            selectDropdownList={comboboxSelectList}
+            selectOnChange={path => {
+              handleUrlPathTypeChange(path)
+            }}
+            selectPlaceholder="Path Scheme"
```

Comment:
The name of the field was not supposed to be renamed :)
```suggestion
            selectPlaceholder="URL"
```